### ISSUE TITLE
Doesn't work on Debian 9

### ISSUE THREAD

**Issue by alexfornuto** [alexfornuto/terminus-installer#2]
```bash
$ php installer.php 

Downloading Terminus 2.0.0 from https://github.com/pantheon-systems/terminus/releases/download/2.0.0/terminus.phar to /tmp 
Moving to /usr/local/bin...
PHP Warning:  rename(/tmp/terminus.phar,/usr/local/bin/terminus.phar): Permission denied in /home/alex/installer.php on line 41

/usr/local/bin requires admin rights to write to...

Download unsuccessful.
```

Compare to Fedora:

```bash
$ php installer.php 

Downloading Terminus 2.0.0 from https://github.com/pantheon-systems/terminus/releases/download/2.0.0/terminus.phar to /tmp 
Moving to /usr/local/bin...
PHP Warning:  rename(/usr/local/bin/terminus.phar): failed to open stream: Permission denied in /home/alex/repos/terminus-installer/installer.php on line 41
PHP Warning:  rename(/tmp/terminus.phar,/usr/local/bin/terminus.phar): Permission denied in /home/alex/repos/terminus-installer/installer.php on line 41

/usr/local/bin requires admin rights to write to...
[sudo] password for alex:
```

**Comment by alexfornuto** [comment]
The script *is* moving the file to `$installdir` using `sudo`, but the function isn't returning `True`.

**Comment by alexfornuto** [comment]
Resolved by #4
